tests/e2e-tests/config/fixture-loaders.js

```diff
@@ -0,0 +1,297 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+const WooCommerceRestApi = require( '@woocommerce/woocommerce-rest-api' )
+	.default;
+
+require( 'dotenv' ).config();
+
+const WooCommerce = new WooCommerceRestApi( {
+	url: `${ process.env.WORDPRESS_BASE_URL }:${ process.env.WORDPRESS_PORT }/`, // Your store URL
+	consumerKey: 'consumer_key', // Your consumer key
+	consumerSecret: 'consumer_secret', // Your consumer secret
+	version: 'wc/v3', // WooCommerce WP REST API version
+	axiosConfig: {
+		auth: {
+			username: process.env.WORDPRESS_LOGIN,
+			password: process.env.WORDPRESS_PASSWORD,
+		},
+	},
+} );
+
+const setupSettings = () => {
+	WooCommerce.post( 'settings/general/batch', {
+		update: [
+			{
+				id: 'woocommerce_store_address',
+				value: '60 29th Street #343',
+			},
+			{
+				id: 'woocommerce_store_city',
+				value: 'San Francisco',
+			},
+			{
+				id: 'woocommerce_store_country',
+				value: 'US:CA',
+			},
+			{
+				id: 'woocommerce_store_postcode',
+				value: '94110',
+			},
+			{
+				id: 'woocommerce_allowed_countries',
+				value: 'specific',
+			},
+			{
+				id: 'woocommerce_specific_allowed_countries',
+				value: [ 'DZ', 'CA', 'NZ', 'ES', 'GB', 'US' ],
+			},
+			{
+				id: 'woocommerce_ship_to_countries',
+				value: 'specific',
+			},
+			{
+				id: 'woocommerce_specific_ship_to_countries',
+				value: [ 'DZ', 'CA', 'NZ', 'ES', 'GB', 'US' ],
+			},
+			{
+				id: 'woocommerce_enable_coupons',
+				value: 'yes',
+			},
+			{
+				id: 'woocommerce_calc_taxes',
+				value: 'yes',
+			},
+			{
+				id: 'woocommerce_currency',
+				value: 'USD',
+			},
+		],
+	} ).catch( ( error ) => console.log( error ) );
+};
+
+const createTaxes = () => {
+	return WooCommerce.post( 'taxes/batch', {
+		create: [
+			{
+				country: 'US',
+				rate: '5.0000',
+				name: 'State Tax',
+				shipping: false,
+				priority: 1,
+			},
+			{
+				country: 'US',
+				rate: '10.000',
+				name: 'Sale Tax',
+				shipping: false,
+				priority: 2,
+			},
+			{
+				country: 'UK',
+				rate: '20.000',
+				name: 'VAT',
+				shipping: false,
+			},
+		],
+	} )
+		.then( ( response ) =>
+			response.data.create.map( ( taxes ) => taxes.id )
```

Comment:
fixtures that are needed later will resolve to an array of newly created things.

Reply:
good point.